src/customize-pop/accessibility-settings.md

```diff
@@ -0,0 +1,125 @@
+
+![Accessibility Settings](/images/accessibility-settings/accessibility-settings.png)
+
+## Always Show Accessibility Menu
+
+When `Always Show Accessibility Menu` is enabled, accessibility options can be toggled using an icon in the upper right corner.
+
+![Always Show Menu](/images/accessibility-settings/always-show-menu.png)
+
+## Seeing
+
+`Seeing` settings provide visual enhancements to help visually impaired users navigate the operating system's UI.
+
+### High Contrast
+
+Enabling `High Contrast` applies a high contrast theme to system windows, application windows, and their icons.
+
+![High Contrast](/images/accessibility-settings/high-contrast.png)
+
+### Large Text
+
+Enabling `Large Text` increases the text size of system windows and applications.
+
+<video autoplay loop>
+    <source src="/images/accessibility-settings/large-text.webm" />
+</video>
+
+### Enable or Disable Animations
+
+Users can enable or disable window animations. Disabling animations can potentially improve performance on system's with low video resources.
+
+<video autoplay loop>
+    <source src="/images/accessibility-settings/disable-animations.webm" />
+</video>
+
+### Cursor Size
+
+`Cursor Size` allows users to choose between five cursor sizes.
+
+![Cursor Size](/images/accessibility-settings/cursor-size.png)
+
+### Zoom
+
+The `Zoom` setting designates all or a part of the screen to display magnified content. The magnified area tracks with the movement of the mouse. This feature can be further customized to zoom specific portions of the screen, increase or decrease the zoom magnification, adding a crosshair, and adding color effects to the zoomed area.
+
+<video autoplay loop>
+    <source src="/images/accessibility-settings/zoom.webm" />
+</video>
+
+### Screen Reader
+
+The Screen Reader option enables a feature that converts the displayed text of a focused window to audible speech.
+
+![Screen Reader](/images/accessibility-settings/screen-reader.png)
+
+### Sound Keys
+
+When `Sound Keys` is enabled, users will hear in audible beep whenever `Num Lock` or `Caps Lock` are turned on or off.
+
+![Sound Keys](/images/accessibility-settings/sound-keys.png)
+
+## Hearing
+
+When the `Visual Alerts` setting is enabled, alert sounds will be accompanied by a visual flickering of the entire screen or window.
+
+<video autoplay loop>
+    <source src="/images/accessibility-settings/visual-alerts.webm" />
+</video>
+
+## Typing
+
+Typing accessibility features enhance keyboard input behavior.
+
+### Screen Keyboard
+
+Enabling `Screen Keyboard` will display an interactive keyboard wherever text entry is possible.
+
+<video autoplay loop>
+    <source src="/images/accessibility-settings/screen-keyboard.webm" />
+</video>
+
+### Repeat Keys
+
+Enabling `Repeat Keys` cause key entry to repeat when a key is held down. Users can configure the delay before this actions occurs, and the repeat speed once the action occurs.
```

Comment:
I would tweak `cause key entry to repeat` to `causes key inputs to repeat`. In the sentence "repeat keys" is singular, as it's a proper noun, leading to "cause" -> "causes" and "entry" -> "entries". And, because "entries" sounded clunky to me in context, I'd switch to "inputs", though that's purely a question of taste.

I'd also highlight that disabling repeat keys will make every key press render a single input somewhere, as the accessibility feature there is really the ability to turn off repeat keys.